.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 16.x]
+        node-version: [12.x, 14.x, 16.x, 18.x]
```

Comment:
Jest 29 will drop 12, fwiw. Might affect integration tests?

Reply:
I think in here we're not yet ready to drop 12.x support. Do you know that this will break?

Reply:
We've stopped testing v12, and setup babel to target v14. Not any explicit breaking changes, tho.

JSDOM v20 also has minimum node@14, fwiw